provisioning/service/samples/getting started/CleanupEnrollmentsSample/CleanupEnrollmentsSample.cs

```diff
@@ -88,7 +89,14 @@ private async Task QueryAndDeleteEnrollmentGroupsAsync()
                     {
                         Console.WriteLine($"Enrollment group to be deleted: {enrollment.EnrollmentGroupId}");
                         s_enrollmentGroupsDeleted++;
-                        await _provisioningServiceClient.EnrollmentGroups.DeleteAsync(enrollment.EnrollmentGroupId);
+                        try
+                        {
+                            await _provisioningServiceClient.EnrollmentGroups.DeleteAsync($"{enrollment.EnrollmentGroupId}_1");
+                        }
+                        catch (ProvisioningServiceException ex) when (ex.StatusCode == HttpStatusCode.NotFound)
```

Comment:
This is added to address pipeline failures. The different frameworks could be attempting the delete the same enrollments, resulting in this error.
Once the tests are updated to clean up their respective resources, we can limit the pipeline cleanup sample to a single target framework run.